pysph/examples/rigid_body/solid_body_floating_in_tank.py

```diff
@@ -0,0 +1,217 @@
+"""A sphere of density 500 falling into a hydrostatic tank (15 minutes)
+
+Check basic equations of SPH to throw a ball inside the vessel
```

Comment:
Is this different from rigid_block_in_tank.py?  If not just retain any one of the correct examples.